hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/StateControllerFactoryTest.java

```diff
@@ -202,7 +202,7 @@ public void build_shouldCreateAuthnFailedErrorController() {
     }
 
     @Test
-    public void build_shouldCreateCountryAuthnFailedErrorStateController() {
+    public void ShouldCreateCountryAuthnFailedErrorStateControllerOnBuild() {
         StateController controller = factory.build(aCountryAuthnFailedErrorState().build(), stateTransitionAction);
```

Comment:
Codacy's going to complain about starting a method with an uppercase letter.

Reply:
Well spotted!! 
Typo, should be small letter of course. Thanks, Kerr!